Sources/MessagingInApp/Gist/Gist.swift

```diff
@@ -8,8 +8,11 @@ protocol GistInstance: AutoMockable {
 }
 
 public class Gist: GistInstance, GistDelegate {
-    var messageQueueManager: MessageQueueManager = DIGraphShared.shared.messageQueueManager
```

Comment:
Changed because `MessageQueueManager` is a singleton now. Use a property getter to get the shared singleton instance.